src/backend/efiling-reviewer-api/README.md

```diff
@@ -11,23 +11,32 @@ This profile mostly enables splunk logging.
 mvn install -P openshift
 ```
 
-### Demo
-
-This profile self isolate the application from any third individual dependencies
-
-```bash
-mvn install -P demo
-```
-
 ## Configuration
 
-You should use environment variables to configure the jag efiling api
-
-| Environment Variable            | Type    | Description                                  | Notes                          |
-| ------------------------------- | ------- | -------------------------------------------- | ------------------------------ |
-| SERVER_PORT                     | Integer | web application server port                  | defaulted to `8080`            |
-| DILIGEN_HEALTH_ENABLED:true}    | Boolean | if Diligen should be monitored on healt check| defaulted to `true`            |
-| DILIGEN_BASE_PATH               | String  | Diligen api base path                        |                                |
+You should use environment variables to configure the jag efiling-reviewer-api
```

Comment:
🤮 I formatted this in notepad++..... Not so much here

Reply:
Accoring to docker-compose, the reviewer-api runs on port 8090